README.md

```diff
@@ -72,9 +72,14 @@ A list of configuration options which you need
   - `ALEXANDRIA_PERMISSION_CLASSES`: Comma-separated list of classes that define permissions for all models
 - Data validation configuration
   - `ALEXANDRIA_VALIDATION_CLASSES`: Comma-separated list of classes that define [custom validations](docs/validation.md)
-
-For development, you can also set the following environemnt variables to help
-you:
+- Thumbnail configuration (optional)
+  - `ALEXANDRIA_ENABLE_THUMBNAIL_GENERATION`: Set to `false` to disable thumbnail generation
+    - Check the docker-compose file for an example on how to set up generation with s3 hooks
+  - `ALEXANDRIA_THUMBNAIL_WIDTH`: Width of generated thumbnails
+  - `ALEXANDRIA_THUMBNAIL_HEIGHT`: Height of generated thumbnails
+  - `ALEXANDRIA_THUMBNAIL_GENERATE_WITHOUT_HOOK`: Set to `true` to generate thumbnails without s3 hooks
```

Comment:
The setting `ALEXANDRIA_THUMBNAIL_GENERATE_WITHOUT_HOOK` is always checked in tandem with `ALEXANDRIA_ENABLE_THUMBNAIL_GENERATION`. I think we could do completely without this setting.

Reply:
Done